packages/react-ui-components/src/stories/Components/TabNavigation.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { ComponentStory } from '@storybook/react';
+import React from 'react';
+import { KvTabNavigation } from '../../components';
+
+// Required to have the correct TagName in the code sample
+KvTabNavigation.displayName = 'KvTabNavigation';
+
+export default {
+	title: 'Components/Tab Navigation',
+	component: 'kv-tab-navigation',
+	argTypes: {
+		selectedTabKey: {
+			control: 'text'
+		}
+	},
+	parameters: {
+		notes: require('@ui-notes/tab-navigation/readme.md')
+	}
+};
+
+const TabNavigationTemplate: ComponentStory<typeof KvTabNavigation> = args => <KvTabNavigation {...args}></KvTabNavigation>;
+
+export const Default = TabNavigationTemplate.bind(this);
```

Comment:
listen the tabChange event and change the tab selected to see the animation working.

Reply:
done ;)